Pod/Classes/ViewControllers/ARArtworkComponentViewController.h

```diff
@@ -0,0 +1,21 @@
+#import "ARComponentViewController.h"
```

Comment:
This is only needed for *root* components, you should only need these type of changes if you're planning to replace `/artworks/:id` for all of artsy

Reply:
This is resolved!